src/main/java/app/coronawarn/verification/model/Tan.java

```diff
@@ -31,7 +31,9 @@
 /**
  * This class represents the transaction number.
  */
-@Schema
+@Schema(
+  description = "The the transaction number model."
```

Comment:
Remove the redundant "the" and everything is fine @dfischer-tech  😉

Reply:
just a little test for reviewers 👍